pkg/generator/generator.go

```diff
@@ -168,6 +168,7 @@ If you find neglible vulnerabilities, print a JSON map formatted and indented as
 	systemPrompt += "* `confidence` (0.0-1.0): higher value if you are confident with the answer.\n"
 	systemPrompt += "* `reason`: the reason why you think the vulnerability is negligible. Should be unique, descriptive, and in 2 or 3 sentences.\n"
 	systemPrompt += "Do not include non-negligible vulnerabilities in the result.\n"
+	systemPrompt += "Only print a valid JSON.\n"
```

Comment:
llms.WithJSONMode() isn't enough?
Maybe this should be reported to https://github.com/tmc/langchaingo/issues

Reply:
OpenAI supports [JSON mode](https://github.com/tmc/langchaingo/blob/4fc43db4fbfb7f9714c7172562fa5b3c5f7583a5/llms/openai/openaillm.go#L114-L116
), but I didn't find it in [Anthoropic](https://github.com/tmc/langchaingo/blob/4fc43db4fbfb7f9714c7172562fa5b3c5f7583a5/llms/anthropic/anthropicllm_option.go).

Reply:
Looks like Cluade itself doesn't have "JSON Mode" for now.
https://github.com/anthropics/anthropic-cookbook/blob/68028f4761c5dbf158b7bf3d43f2f45b44111200/misc/how_to_enable_json_mode.ipynb